### ISSUE TITLE
Further structure formats

### ISSUE THREAD

**Issue by h-2** [seqan/seqan3#309]
After #307 is merged, this is a meta ticket for further RNA formats:

  - [x] vienna (part of #307)
  - [ ] stockholm 1
  - [ ] connect

Should we add the following?
  - [ ] stockholm 2
  - [ ] ...

Here some links:
  * http://projects.binf.ku.dk/pgardner/bralibase/RNAformats.html
  * http://www.ibi.vu.nl/programs/k2nwww/static/data_formats.html

**Comment by joergi-w** [comment]
> Should we add the following?
> stockholm 2

Yes, the Stockholm structured alignment format is important, because it is the common format in the Rfam and Pfam data bases (and probably I'm going to use it as output format of SeqAn3's LaRA). However, we probably do not need Stockholm 1.
